doc_source/CNAMEs.md

```diff
@@ -150,11 +152,17 @@ You can’t add a wildcard to a top level domain name, such as `*.com`, so if yo
    ```
 
    The line in the Answer Section shows a CNAME resource record set that routes queries for images\.example\.com to the CloudFront distribution domain name d111111abcdef8\.cloudfront\.net\. The CNAME resource record set is configured correctly if the name on the right side of `CNAME` is the domain name for your CloudFront distribution\. If that is any other value, for example, the domain name for your Amazon S3 bucket, then the CNAME resource record set is configured incorrectly\. In that case, go back to Step 4 and correct the CNAME record to point to the domain name for your distribution\.
+**Note**  
+When using Route 53 alias records, it is not possible to use dig to confirm that the resource record points to the new distribution\. In this case, you can either change the type of of the resource record from alias to CNAME, or wait until the record's time to live \(TTL\) value has expired\.
 
-1. Remove the CNAME from the existing distribution and move it to the new CloudFront distribution\. For example, move `marketing.example.com` to a new distribution that by default is pointed to by something like `d111111abcdef8.cloudfront.net`\.
+1. Remove the CNAME from the existing distribution and then add it to the new CloudFront distribution where the wildcard alternate name was added previously\.
+**Note**  
+While these changes propagate, the alternate domain name may be served by either the original or new distribution at random\. This behaviour may even persist for a few minutes after both distributions have reached status **Deployed**\.
```

Comment:
We use the US spelling ("behavior", not "behaviour") in AWS documentation.

Reply:
Addressed with fixup commit, that will auto-squash correctly with `git rebase --autosquash <ref>`

Also removed the "even" word there – rationale in commit message.